build/index.html

```diff
@@ -544,63 +544,291 @@ <h2>Govcy Inputs</h2>
                 <div class="govcy-form-control">
-                    <label class="govcy-label" for="">Postal code</label>
-                    <input id="" type="text" class="govcy-text-input" placeholder="Postal code" />
+                    <label class="govcy-label govcy-mb-1" for="">Date of birth</label>
+                    <span class="govcy-hint govcy-mb-3">For emample&#44; 12 11 2007</span>
+                    <div class="govcy-d-flex">
+                        <div class="govcy-d-block">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Day</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_3 govcy-p-2" placeholder="" maxlength="2" />
+                        </div>
+                        <div class="govcy-d-block govcy-ml-2">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Month</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_3 govcy-p-2" placeholder="" maxlength="2" />
+                        </div>
+                        <div class="govcy-d-block govcy-ml-2">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Year</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_6 govcy-p-3" placeholder="" maxlength="4" />
+                        </div>
+                    </div>
+                </div>
+            </fieldset>
+        </form>
+
+        <form action="" class="govcy-form">
+            <fieldset class="govcy-fieldset">
+                <div class="govcy-form-control govcy-form-control-error">
+                    <label class="govcy-label govcy-mb-1" for="">Date of birth</label>
+                    <span class="govcy-input-error-msg govcy-mb-3">There is an error</span>
+                    <div class="govcy-d-flex">
+                        <div class="govcy-d-block">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Day</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_3 govcy-p-2" placeholder="" maxlength="2" />
+                        </div>
+                        <div class="govcy-d-block govcy-ml-2">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Month</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_3 govcy-p-2" placeholder="" maxlength="2" />
+                        </div>
+                        <div class="govcy-d-block govcy-ml-2">
+                            <label class="govcy-label govcy-mb-1 govcy-fw-normal govcy-mb-2" for="">Year</label>
+                            <input type="text" class="govcy-text-input govcy-text-input-char_6 govcy-p-3" placeholder="" maxlength="4" />
```

Comment:
the width of the year is too big. I checked the Percy screenshoots and something went wrong when you made the updates on the checkbox and radio border. See the difference in the year button from the [older](https://percy.io/a0d81626/dsf-design-system/builds/15190276/changed/858208104?browser=edge&browser_ids=18%2C19%2C20%2C21&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=new&width=1280&widths=375%2C767%2C1280) screenshoot and the [latest](https://percy.io/a0d81626/dsf-design-system/builds/15191146/changed/858264121?browser=edge&browser_ids=18%2C19%2C20%2C21&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=new&width=1280&widths=375%2C767%2C1280)